action.yml

```
@@ -315,6 +334,14 @@ runs:
         PATH=$PATH:$(pwd)
         cd $GITHUB_WORKSPACE
         digger
+
+    - uses: actions/cache/save@v4
+      name: cache-save
+      with:
+        path: ${{ github.workspace }}/cache
+        key: digger-${{ hashFiles('**/cache') }}
```

```suggestion
        key: digger-cache-${{ hashFiles('**/cache') }}
```

Think we might need it this way, as digger-${{ hashFiles }} would not be a match for digger-cache in any way, but digger-cache-abc123 should be a general match